modules/images-update-global-pull-secret.adoc

```diff
@@ -25,18 +25,43 @@ As of {product-title} 4.7, changes to the global pull secret no longer trigger a
 
 .Prerequisites
 
-* You have a new or modified pull secret file to upload.
 * You have access to the cluster as a user with the `cluster-admin` role.
 
+
 .Procedure
+To update the global pull secret, complete the following steps:
+[NOTE]
+====
+To overwrite your existing pull secret using a new pull secret file, skip steps 1 & 2.
+====
+
+. Download the pull secret:
++
+[source,terminal]
+----
+$ oc get secret/pull-secret -n openshift-config --template='{{index .data ".dockerconfigjson" | base64decode}}' ><pull_secret_location> <1>
+----
+<1> Provide the path to the pull secret.
+
+. Add the new pull secret:
++
+[source,terminal]
+----
+$ oc registry login --registry="<registry>" \ <1>
+--auth-basic="<username>:<password>" \ <2>
+--to=<pull-secret-location>
+----
+<1> Provide the new registry.
+<2> Provide the credentials of your new registry.
++
+Alternatively, you can perform a manual update to the pull secret file.
 
-* Enter the following command to update the global pull secret for your cluster:
+. Update the global pull secret for your cluster:
```

Comment:
We're actually discussing making the text that you're removing required. The discussion is trending toward requiring it, so I suggest restoring this line.

Reply:
@kalexand-rh Do I have to mention this for every step i.e. 'enter the following command'? I removed this to keep it consistent with other steps since adding it in other steps looked quite repetitive. How about having a global message on top that they have to run the following commands?

Reply:
Appending it on every step is what we're leaning toward - it's an accessibility nightmare to just start the command.

Reply:
@kalexand-rh I have made all the suggested changes. Kindly review.